README.md

```diff
@@ -39,45 +39,87 @@ evaluation purposes.
-2) Compile it with `make build`. You need a golang 1.19 environment to do it. make build will run a go install so
+1) Compile it with `make build`. You need a golang 1.19 environment to do it. make build will run a go install so
 it will generate a binary for each of the commands:
     - platform
     - migrate
     - pending_publisher
     - configurator
-3) Make sure you have postgres, redis and hashicorp vault properly configured. You could use `make up` command to start
+2) Make sure you have postgres, redis and hashicorp vault properly configured. You could use `make up` command to start
 a postgres, redis and vault redis container. Use this images only for evaluation purposes.
-4) Make sure that your database is properly configured (step 1) and run `make db/migrate` command. This will check for the
+3) Make sure that your database is properly configured (step 1) and run `make db/migrate` command. This will check for the
 current structure of the database and will apply needed change to create or update the database schema.
-5) Write the vault token in the config.toml file, once the vault is initialized the token can be found in _infrastructure/local/.vault/data/init.out_ or in the logs of the vault container.
-6) Run `./bin/platform` command to start the issuer. Browse to http://localhost:3001 (or the port configured in ServerPort config entry)
+4) Write the vault token in the config.toml file, once the vault is initialized the token can be found in _infrastructure/local/.vault/data/init.out_ or in the logs of the vault container.
+5) Run `./bin/platform` command to start the issuer. Browse to http://localhost:3001 (or the port configured in ServerPort config entry)
 This will show you the api documentation.
-7) Run `./bin/pending_publisher` in background. This process is not strictly necessary but highly recommended. 
+6) Run `./bin/pending_publisher` in background. This process is not strictly necessary but highly recommended. 
 It checks for possible errors publishing transactions on chain and try to resend it.
-8) Follow the [steps](#steps-to-write-the-private-key-in-the-vault) to write the private key in the vault
+7) Follow the [steps](#steps-to-write-the-private-key-in-the-vault) to write the private key in the vault
+8) Set up [required environment variables](#how-to-configure)
 
 
 ## How to configure
-The server can be configured with a config file and/or environment variables. There is a config.toml.sample file provided
-as a reference. The system expects to have a config.toml file in the working directory. 
+The different components must be configured with environment variables. The following are the environment variables required with some sample values, and we encourage you to split them into
+three files. Those three files are actually used by the [docker-compose](infrastructure/local/docker-compose.yml) file provided.
+
+```
+# .env file variables
+SH_ID_PLATFORM_NATIVE_PROOF_GENERATION_ENABLED=true
```

Comment:
that prefix is the name of the project. In an environment like ecs on AWS with a tons of variables could be useful the prefix. What prefix make sense to you? @cneuro

Reply:
It's a good point. My main concern is that it isn't "platform". Now it might be good time as any to call it "issuer node"? So perhaps `SH_ID_ISSUER_`, at least it's a bit shorter?

Reply:
It sounds good.

Reply:
Talking we BE guys we decided change to just `ISSUER` prefix